src/sentry/incidents/logic.py

```diff
@@ -396,30 +414,36 @@ def bulk_get_incident_aggregates(query_params_list):
     return [result["data"][0] for result in results]
 
 
-def bulk_get_incident_stats(incidents, prewindow=False):
+def bulk_get_incident_stats(incidents, windowed_stats=False):
     """
     Returns bulk stats for a list of incidents. This includes unique user count,
     total event count and event stats.
-    Note that even though this function accepts a prewindow parameter, it does not
-    affect the snapshots if they were created using a prewindow. Only the live fetched stats.
+    Note that even though this function accepts a windowed_stats parameter, it does not
+    affect the snapshots. Only the live fetched stats.
     """
-    closed = [i for i in incidents if i.status == IncidentStatus.CLOSED.value]
     incident_stats = {}
-    snapshots = IncidentSnapshot.objects.filter(incident__in=closed)
-    for snapshot in snapshots:
-        event_stats = snapshot.event_stats_snapshot
-        incident_stats[snapshot.incident_id] = {
-            "event_stats": SnubaTSResult(
-                event_stats.snuba_values, event_stats.start, event_stats.end, event_stats.period
-            ),
-            "total_events": snapshot.total_events,
-            "unique_users": snapshot.unique_users,
-        }
+    # closed = [i for i in incidents if i.status == IncidentStatus.CLOSED.value]
```

Comment:
Is this just temporary, or do we want to remove entirely?

Reply:
Just temporary, I wanted to be able to see the new windowed calculations in staging without being shown the previously taken snapshots.

Reply:
I think we're going to end up with weird interactions here based on how the snapshot is created